packages/block-library/src/navigation-area/edit.js

```diff
@@ -51,6 +52,10 @@ function NavigationAreaBlock( { attributes, setAttributes } ) {
 		[ navigationAreas ]
 	);
 
+	deprecated( 'wp.blockLibrary.NavigationArea', {
+		since: '5.9',
```

Comment:
Has this block ever landed on Core as stable? If not, we should use a Gutenberg version here and specify a `plugin: "gutenberg"` here as well

Reply:
> Has this block ever landed on Core as stable

Not it's not in a stable release of Core only Gutenberg Plugin. I'll spin up a follow up unless @adamziel gets there first.

Thanks for letting us know.

Reply:
@youknowriad Here is the followup https://github.com/WordPress/gutenberg/pull/36777